homeassistant/components/stream/hls.py

```diff
@@ -95,11 +94,12 @@ def render_playlist(track):
             # at the beginning or we risk a behind live window exception in exoplayer.
             # EXT-X-START is not supposed to be within 3 target durations of the end,
             # but this seems ok
-            f"#EXT-X-START:TIME-OFFSET=-{EXT_X_START * track.target_duration:.3f},PRECISE=YES",
+            f"#EXT-X-START:TIME-OFFSET=-{EXT_X_START * track.target_duration:.3f}",
```

Comment:
PRECISE=YES was removed. Without PRECISE specified (or with PRECISE=NO), players start from the beginning of the segment where the time offset is found. This makes more sense for now since we are only delivering whole segments.

Reply:
I'll stick a comment in there noting this.